deploy/kubernetes/helm-charts/altinn-runtime/templates/ingress.yaml

```diff
@@ -0,0 +1,26 @@
+{{- $serviceName := include "altinn-runtime.fullname" . }}
+{{- $containerPort := .Values.image.containerPort }}
```

Comment:
A deployment/pod might map to several ports, this limits us to only using one port per deployment/pod.

Reply:
Keeping simple until we need feature.